model_compiler/python/nnir_to_clib.py

```diff
@@ -1975,7 +1975,7 @@ def main():
     inputFolder = sys.argv[pos]
     outputFolder = sys.argv[pos+1]
     print('reading IR model from ' + inputFolder + ' ...')
-    graph = IrGraph()
+    graph = IrGraph(True)
```

Comment:
Codacy found an issue: ['IrGraph' may be undefined, or defined from star imports: nnir (F999)](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=5754497)